pkg/grpc/admin/proto/admin.proto

```diff
@@ -234,6 +234,146 @@ service AdminService {
             permission: "iam.write"
         };
     }
+
+    rpc IdpByID(IdpID) returns (IdpView) {
+        option (google.api.http) = {
+            get: "/idps/{id}"
+        };
+
+        option (caos.zitadel.utils.v1.auth_option) = {
+            permission: "iam.idp.read"
+        };
+    }
+
+    rpc CreateOidcIdp(OidcIdpConfigCreate) returns (Idp) {
+        option (google.api.http) = {
+            post: "/oidcidps"
```

Comment:
isn't oidc a subtype of idps? I have the same problem in my human/machine branch. In my opinion it should be idps/oidc and idps/saml

Reply:
we have it like this in applications, but its ok for me. i will change it in both requests.